src/App.tsx

```diff
@@ -1,12 +1,89 @@
 import React from 'react';
+
+import { GoodList } from './components/GoodsList';
 import './App.scss';
 
 // import { getAll, get5First, getRed } from './api/goods';
 // or
-// import * as goodsAPI from './api/goods';
+import * as goodsAPI from './api/goods';
```

Comment:
better to import functions
```suggestion
import {getAll, get5First, getRedGoods} from './api/goods';
```

Reply:
`import * as goodsAPI from './api/goods';`
syntax like this can prevent some syntax mistakes, am I right?